### ISSUE TITLE
Need a cross-platform @Nonnull

### ISSUE THREAD

**Issue by jeffallen** [dedis/cothority#1328]
We needed to remove `@Nonnull` to make Cothority compile on Android.

It would be nice to be able to continue using source annotations, but we need them to be cross platform.

The task is to re-add the `@Nonnull` annotations back in, but in whatever way is normal for Android and Java cross-compatibility.

**Comment by kc1212** [comment]
fixed in https://github.com/dedis/cothority/pull/1321/

**Comment by jeffallen** [comment]
This issue was a reminder to find a better solution that nuking them. Reopening.

**Comment by kc1212** [comment]
Right, sorry about that. There seems to be a framework that can do null check https://checkerframework.org/ this is where i found the information https://stackoverflow.com/questions/4963300/which-notnull-java-annotation-should-i-use
